tools/launch.ts

```diff
@@ -147,6 +152,10 @@ const relays: { [name: string]: NetworkConfig } = {
     docker: "purestake/moonbase-relay-testnet:sha-7da182da",
     chain: "rococo-local",
   },
+  "rococo-9112": {
+    docker: "purestake/moonbase-relay-testnet:sha-415ce0db",
```

Comment:
According to @crystalin, this image contains rococo-9106. Can you double check ?

Reply:
You can re-use `rococo-9111` I think, it is fine

Reply:
Thank you, yeah it seems that in `0.9.11` release rococo runtime version was `9111`, but it went back to `9106` in polkadot `0.9.12` release
https://github.com/paritytech/polkadot/blob/release-v0.9.12/runtime/rococo/src/lib.rs#L109